paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsActivityTest.kt

```diff
@@ -263,16 +263,16 @@ internal class PaymentOptionsActivityTest {
         val args = PAYMENT_OPTIONS_CONTRACT_ARGS.updateState(isGooglePayReady = true)
         val viewModel = createViewModel(args)
 
-        val transitionTargets = mutableListOf<TransitionTarget>()
-        viewModel.transition.observeEventsForever { transitionTargets.add(it) }
+        val transitionTargets = mutableListOf<TransitionTarget?>()
+        viewModel.currentScreen.asLiveData().observeForever { transitionTargets.add(it) }
```

Comment:
Can we remove the `asLiveData` and use turbine for these tests instead?

Reply:
Yep, for sure!